platform_as_a_service.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Platform as a Service (PaaS)
+status: Completed
+category: Tecnologia
+---
+
+## Cos'è
+Una Platform as a Service, o PaaS, è una piattaforma esterna ai team di sviluppo per distribuire ed eseguire le proprie applicazioni. Heroku, Cloud Foundry, App Engine sono esempi di offerte PaaS.
+
+## Quali problematiche affronta
+Per sfruttare i modelli nativi del cloud come i [microservizi](/microservizi/) o le applicazioni distribuite, i team delle operation e gli sviluppatori devono essere in grado di scaricare dalle proprie board una quantità significativa di operazioni e lavori di manutenzione. Questi includono attività come il provisioning dell'infrastruttura, la gestione dell'individuazione dei servizi e il bilanciamento del carico, e la scalabilità delle applicazioni.
```

Comment:
This link needs to go to [the English version of `Microservices`](https://glossary.cncf.io/microservices/). 😊

```suggestion
Per sfruttare i modelli nativi del cloud come i [microservizi](/microservices/) o le applicazioni distribuite, i team delle operation e gli sviluppatori devono essere in grado di scaricare dalle proprie board una quantità significativa di operazioni e lavori di manutenzione. Questi includono attività come il provisioning dell'infrastruttura, la gestione dell'individuazione dei servizi e il bilanciamento del carico, e la scalabilità delle applicazioni.
```